lib/http.rb

```diff
@@ -15,7 +15,17 @@
 require "http/response/body"
 require "http/response/parser"
 
-# HTTP should be easy
+# HTTP should be easy.
+#
+# Basic Usage
+#
+# HTTP.post("https://server.com", options)
+#
+# For more detailed documentation and usage notes:
```

Comment:
I think this line is redundant a bit and with `@see` tag below it will be rendered a bit bad..
```suggestion
```